spock/pom.xml

```diff
@@ -19,6 +19,7 @@
         <dependency>
             <groupId>org.spockframework</groupId>
             <artifactId>spock-core</artifactId>
+            <version>2.3-groovy-3.0</version>
```

Comment:
Could you please update this version in the root pom.xml by changing the property `spockframework.version`? It's not neccessary to define exact version here